nodejs/eks/cluster.ts

```diff
@@ -620,6 +717,14 @@ export class Cluster extends pulumi.ComponentResource {
      */
     public readonly instanceRoles: pulumi.Output<aws.iam.Role[]>;
 
+    /**
+     * The IAM Role used to create & authenticate against the EKS cluster. This role is given `[system:masters]`
+     * permission in K8S, See: https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html
+     *
+     * Note: When supplying a role, make sure that it has `eks:*` and `iam:Passrole` permissions.
```

Comment:
```suggestion
     * Note: When supplying a role, make sure that it has permissions for each action, `eks:*` and `iam:Passrole`,  on `Resource: *`.
```

Consider adding a bit more detail on the requirements.